### PR TITLE
Docs for con types

### PR DESCRIPTION
**Pull request by tayoogunbiyi**
## Change Summary

This change adds documentation for the con* type functions

## Related issue number
closes https://github.com/samuelcolvin/pydantic/issues/2242

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)

### CODE REVIEW

File: docs/usage/types.md [hunk 1]
```diff
@@ -722,6 +722,56 @@ _(This script is complete, it should run "as is")_
...
+- `lt: Decimal = None`: enforces decimal to be less than the set value
+- `le: Decimal = None`: enforces decimal to be less than or equal to the set value
+- `max_digits: int = None`: maximum number of digits within the decimal. it does not include a zero before the decimal point or trailing decimal zeroes
+- `decimal_places: int = None`: max number of decimal places allowed. it does not include trailing decimal zeroes
+- `multiple_of: Decimal = None`: enforces decimal to be a multiple of the set value
+
+
+
+
+
```
Comment (PrettyWood) [inline docs/usage/types.md:774]:
```suggestion
```

Comment (PrettyWood) [inline docs/usage/types.md:774]:
```suggestion
```

**Review by PrettyWood — COMMENTED** [review]
Thanks for that!

**Comment by samuelcolvin** [comment]
this is great, thank you so much.

**Pull request merged**